model-hub/model_hub.json

```diff
@@ -0,0 +1,92 @@
+[
+  {
+    "id": "mobilenetv2",
+    "description": "An efficient image classification model, pretrained on ImageNet dataset which contains images from 1,000 classes.",
+    "task": "classification",
```

Comment:
I will suggest that we use the value of protobuf like `TASK_CLASSIFICATION`

Reply:
Done